python/ray/_private/structured_logging/utils.py

```diff
@@ -0,0 +1,55 @@
+from ray._private.structured_logging.formatters import TextFormatter
+from ray._private.structured_logging.filters import CoreContextFilter
+from typing import Union
+
+# A dictionary to map encoding types to their corresponding logging configurations.
+LOG_MODE_DICT = {
+    "TEXT": {
+        "version": 1,
+        "disable_existing_loggers": False,
```

Comment:
> disable_existing_loggers – If specified as False, loggers which exist when this call is made are left enabled. The default is True because this enables old behaviour in a backward-compatible way. This behaviour is to disable any existing non-root loggers unless they or their ancestors are explicitly named in the logging configuration.

https://docs.python.org/3/library/logging.config.html